integrations/deepl.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: integration
+name: DeepL
+description: Use DeepL translation services with Haystack
+authors:
+    - name: Dribia Data Research
+      socials:
+        github: dribia
+        linkedin: https://www.linkedin.com/company/dribia
+pypi: https://pypi.org/project/deepl-haystack/
+repo: https://github.com/dribia/deepl-haystack
+type: Custom Component
+report_issue: https://github.com/dribia/deepl-haystack/issues
+logo: /logos/deepl.png
+version: Haystack 2.0
+toc: true
+---
+### **Table of Contents**
+- [Overview](#overview)
+- [Installation](#installation)
+- [Usage](#usage)
+  - [Components](#components)
+- [Examples](#examples)
+  - [Standalone](#standalone)
+  - [Pipeline](#pipeline)
+- [License](#license)
+
+## Overview
+
+[DeepL](https://www.deepl.com/) is a powerful translation services provider, offering high-quality translations 
+in multiple languages. This integration allows you to use DeepL's translation services with Haystack. 
+
+## Installation
+
+```console
+pip install deepl-haystack
+```
+
+## Usage
```

Comment:
Here or below, could you mention that the user should export the `DEEPL_API_KEY` environment variable? We are trying to recommend environment variables as the way to deal with api keys 🙏 
That way, you could remove the `.from_token` in the initialization code below

Reply:
Just pushed a new version of the file without `Secret` usage while suggesting env var usage, which I agree is a better practice :+1: